### ISSUE TITLE
version extensions issue.

### ISSUE THREAD

**Issue by osonoi** [IBM/drupal-on-kubernetes-sample#27]
Whan I try to create, I got following error. Need to change yml file?

$ kubectl create -f kubernetes/postgres.yaml
service/postgresql created
persistentvolumeclaim/postgres-claim created
error: unable to recognize "kubernetes/postgres.yaml": no matches for kind "Deployment" in version "extensions/v1beta1"

**Comment by AnthonyAmanse** [comment]
yup - these yml files need some updates. These were made when kubernetes version was < v1.9.0
